### PR TITLE
feat(RESTJSONErrorCodes): add error `30034`

### PR DESCRIPTION
**Pull request by Mateo-tem**
**Please describe the changes this PR makes and why it should be merged:**

**If applicable, please reference Discord API Docs PRs or commits that influenced this PR:**

- https://github.com/discord/discord-api-docs/pull/5200

### CODE REVIEW

File: rest/common.ts [hunk 1]
```diff
@@ -103,7 +103,7 @@ export enum RESTJSONErrorCodes {
 	GuildAlreadyHasTemplate = 30031,
 	MaximumNumberOfApplicationCommandsReached,
 	MaximumThreadParticipants,
-
+	MaximumDailyApplicationCommandCreates,
 	MaximumNumberOfNonGuildMemberBansHasBeenExceeded = 30035,
```
Comment (vladfrangu) [inline rest/common.ts:107]: This = is no longer needed then

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: rest/common.ts (hunk 1 above)
Comment (Mateo-tem) [reply rest/common.ts:107]: huh? why?

Comment (vladfrangu) [reply rest/common.ts:107]: For the same reason you don't need it for MaximumNumberOfApplicationCommandsReached, MaximumThreadParticipants and now MaximumDailyApplicationCommandCreates. It autoincrements

Comment (Mateo-tem) [reply rest/common.ts:107]: Oh, right. I will close this, thanks.

Comment (vladfrangu) [reply rest/common.ts:107]: Thats not what I meant! This error needs to be documented, its just that MaximumNumberOfNonGuildMemberBansHasBeenExceeded doesn't need to be explicitly declared as 30035!

Comment (ImRodry) [reply rest/common.ts:107]: Pretty sure he meant the value isn’t needed, no reason to close the PR 🤔

Comment (Mateo-tem) [reply rest/common.ts:107]: Oh, so sorry. I misread all 🤦‍♂️. I should have removed that before, I will change that!

Comment (Mateo-tem) [reply rest/common.ts]: Done!

**Review by vladfrangu — APPROVED** [review]

File: rest/common.ts [hunk 2]
```diff
@@ -103,8 +103,8 @@ export enum RESTJSONErrorCodes {
 	GuildAlreadyHasTemplate = 30031,
 	MaximumNumberOfApplicationCommandsReached,
 	MaximumThreadParticipants,
-
-	MaximumNumberOfNonGuildMemberBansHasBeenExceeded = 30035,
+	MaximumDailyApplicationCommandCreates,
```
Comment (vvito7) [inline rest/common.ts:106]:
```suggestion
	MaximumDailyApplicationCommandCreatesReached,
```
think it's more understandable? i would do it even for the one above

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**